quickstatements/latest/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# Test if required environment variables have been set
+REQUIRED_VARIABLES=(OAUTH_CONSUMER_KEY OAUTH_CONSUMER_SECRET QS_PUBLIC_HOST_AND_PORT WB_PUBLIC_HOST_AND_PORT WIKIBASE_HOST)
+for i in ${REQUIRED_VARIABLES[@]}; do
+    eval THISSHOULDBESET=\$$i
+    if [ -z "$THISSHOULDBESET" ]; then
```

Comment:
Without icky `eval`:

```sh
if ! [[ -v $i ]]; then
```

If you want to test nonemptiness as well:

```sh
if ! [[ -v $i && -n ${!i} ]]; then
```

Reply:
Nice! This is so much cleaner. I didn't know bash's extended test had -v for exactly this.

I think it's best to test for set and still pass on emptyness in case the user wants to do something like run with broken OAuth keys because they want to hand craft their config.json after running